modular_zubbers/code/modules/vending/megaseed.dm

```diff
@@ -0,0 +1,6 @@
+/obj/machinery/vending/hydroseeds/Initialize(mapload)
+	for(var/category in product_categories)
+		if(category["name"] == "Fruits")
+			category["products"] += list(/obj/item/seeds/rockfruit = 3)
```

Comment:
This feels pretty ugly but it is what it is.

Reply:
Can't do it much better unfortunately